module/userExtension/validator/v1.js

```diff
@@ -0,0 +1,15 @@
+module.exports = (req) => {
+
+    let userExtensionValidator = {
```

Comment:
@deepakr123 , @rakeshSgr - Please add request validations for all API endpoints

Get query parameters, post body parameters all should be validated which si required and type checking to be done

Reply:
resolved

Reply:
resolved